music21/sorting.py

```diff
@@ -31,14 +31,14 @@
 from math import inf as INFINITY
 from music21 import exceptions21
 
-_attrList = ['atEnd', 'offset', 'priority', 'classSortOrder', 'isNotGrace', 'insertIndex']
-
 
 class SortingException(exceptions21.Music21Exception):
     pass
 
 
-class SortTuple(namedtuple('SortTuple', _attrList)):
+class SortTuple(namedtuple('SortTuple', (
+    'atEnd', 'offset', 'priority', 'classSortOrder', 'isNotGrace', 'insertIndex'
+))):
```

Comment:
Somewhere on an issue mypy/guido said they're never going to support dynamic namedtuple fields.

Reply:
<s>Can _attrList be renamed '_sortAttributes' and be made a tuple and then consulted below? This is so performance critical that I don't want to have the extra method resolution step in the methods below</s> -- never mind: the methods below are rarely called, so it's okay.  Still I think it's somewhat clearer than self.fields.